_layouts/call.html

```diff
@@ -0,0 +1,55 @@
+<!-- <div class="ui container divider"></div> -->
+
+<div id="{{ page.priority }}" class="ui {{ page.color | default: 'blue' }} grid aligned segment call">
+    <div class="one wide column"></div>
+
+    {% if page.tag == 'left' %}
```

Comment:
Bad property name. What does it mean that a `tag` is `left` or `right`? I expect a tag to be some kind of _keyword_, not a _side_.

I doubt the YAML front-matter should contain positioning information.